docs/Using-Fleet/Adding-hosts.md

```diff
@@ -211,15 +214,15 @@ Test the installers on each platform before initiating the migration.
 
 #### Migrate
 
-Using your standard deployment tooling (Chef, Puppet, etc.), install the generated package. At this
+Install the generated package using your standard deployment tooling (Chef, Puppet, etc.). At this
 time, [uninstall the existing
 osquery](https://blog.fleetdm.com/how-to-uninstall-osquery-f01cc49a37b9).
 
 If the existing enrolled hosts use `--host_identifier=uuid` (or the `uuid` setting for Fleet's
 [osquery_host_identifier](../Deploying/Configuration.md#osquery-host-identifier)), the new
 installation should appear as the same host in the Fleet UI. If other settings are used, duplicate
 entries will appear in the Fleet UI. The older entries can be automatically cleaned up with the host
-expiration functionality configured in the application settings (UI or fleetctl).
+expiration functionality configured in the application settings (UI or fleetctl). 
```

Comment:
Should "application settings" be capitalized?

Reply:
Ah, good catch. I think we need to update this to walk the user through what buttons to select to locate this setting.

Can you please update "application settings" to "**Settings > Organization settings > Advanced**"?